manifests/kadalu-operator-microk8s.yaml

```diff
@@ -282,7 +283,11 @@ spec:
               value: "kadalu"
             - name: KADALU_VERSION
               value: "devel"
+            - name: KADALU_NAMESPACE
+              value: "kadalu"
             - name: KUBELET_DIR
               value: "/var/snap/microk8s/common/var/lib/kubelet"
             - name: K8S_DIST
-              value: "microk8s"
\ No newline at end of file
+              value: "microk8s"
+            - name: VERBOSE
+              value: "False"
```

Comment:
@amarts just noticed that, `"False"` here will result in adding verbose logs because all env variables are passed as string.  https://github.com/kadalu/kadalu/blob/devel/lib/kadalulib.py#L139

We should change this to `VERBOSE = yes|no` and compare the same inside kadalulib.py

Sorry @leelavg, issue is not related to your PR. But noticed the issue here.

Reply:
@aravindavk np, while raising this PR i noticed two other minor issues and added the fix. Actually I wanted to bring this to your notice coz, True/False in helm template is being forced to behave a string because of VERBOSE env value.

I can rebase once the value is changed and committed.

Reply:
Fixed.